packages/round-manager/src/features/common/CopyToClipboardButton.tsx

```diff
@@ -1,24 +1,32 @@
 import { Button } from "./styles";
 import { ClipboardCopyIcon } from "@heroicons/react/solid";
+import { useState } from "react";
 
 type CopyToClipboardType = {
   textToCopy: string;
-  clipboardText?: string;
   styles?: string;
   iconStyle?: string;
 };
 
 export default function CopyToClipboardButton(props: CopyToClipboardType) {
+
+  const [active, setActive] = useState(false);
+
   return (
+
     <Button
       type="button"
-      className={`inline-flex ${props.styles}`}
-      onClick={async () =>
-        await navigator.clipboard.writeText(props.textToCopy)
-      }
+      className={`inline-flex bg-violet-100 text-violet-600 ${active ? 'animate-[violetTransition_20s_ease-in]' : ''} ${props.styles}`}
+      onClick={async () => {
+        setActive(true);
+        setTimeout(() => setActive(false), 1000);
```

Comment:
super nitpicky, but 1sec still feels too fast to me? would be good to ask Melissa to take a quick look

Reply:
Fair ! I'll throw this up on staging and ask melissa to check this out